orbit/db2.py

```diff
@@ -1,5 +1,5 @@
 import config
-from peewee import Model, IntegerField, CharField, SqliteDatabase
+from peewee import Model, IntegerField, CharField, SqliteDatabase, FieldAccessor  # noqa 501
```

Comment:
for consistency with existing lint disable comments in the repo, it should be `# NOQA: E501`